packages/tempus/src/Tempus_StepperIMEX_RK_Partition_decl.hpp

```diff
@@ -417,7 +421,8 @@ class StepperIMEX_RK_Partition : virtual public Tempus::StepperImplicit<Scalar>,
     virtual OrderODE getOrderODE()   const {return FIRST_ORDER_ODE;}
   //@}
 
-  Teuchos::RCP<Thyra::VectorBase<Scalar> >& getStageZ() {return stageZ_;};
+  Teuchos::RCP<Thyra::VectorBase<Scalar> > getStageX() {return stageZ_;}
+  Teuchos::RCP<Thyra::VectorBase<Scalar> > getStageZ() {return stageZ_;};
```

Comment:
Can we doxygen on this?

Reply:
Yeah, sounds like I good idea.  I mentioned it in the commit, but something more permanent would be a good idea.

Reply:
Done!